### PR TITLE
Tim/gtm 57 updated landing page

### PR DESCRIPTION
**Pull request by tg339**

### CODE REVIEW

File: apps/framework-landing/components.json
Comment (georgevanderson) [inline apps/framework-landing/components.json:1]: Is this needed? If shadcn components are in design system

File: apps/web/components.json
Comment (georgevanderson) [inline apps/web/components.json:1]: Same question as framework-landing components.json, but I assume this whole directory will be deleted

File: apps/framework-landing/gsap-bonus.tgz
Comment (georgevanderson) [inline apps/framework-landing/gsap-bonus.tgz:1]: Can this be deleted since in design-system

File: apps/web/components.json
Comment (tg339) [reply apps/web/components.json:1]: yeah. Nuking this whole directory once I'm done

File: apps/create-moose-app/tsconfig.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "tsconfig/base.json",
+  "extends": "@repo/ts-config/base.json",
```
Comment (callicles) [inline apps/create-moose-app/tsconfig.json:2]: Why the `@repo` added?

File: apps/company-landing/package-lock.json
Comment (callicles) [inline apps/company-landing/package-lock.json:1]: Should be removed :)

File: apps/framework-landing/tsconfig.json [hunk 2]
```diff
@@ -1,32 +1,9 @@
 {
+  "extends": "@repo/ts-config/nextjs.json",
```
Comment (callicles) [inline apps/framework-landing/tsconfig.json:2]: why the rename from `ts-config` to `@repo/ts-config`?

File: apps/moose-console/next.config.mjs [hunk 3]
```diff
@@ -51,6 +54,6 @@ const sentryWebpackPluginOptions = {
 };
 
 // Make sure adding Sentry options is the last code to run before exporting
-module.exports = nextConfig;
+// export default nextConfig;
 
-// withSentryConfig(nextConfig, sentryWebpackPluginOptions);
+export default withSentryConfig(nextConfig, sentryWebpackPluginOptions);
```
Comment (callicles) [inline apps/moose-console/next.config.mjs]: This breaks the build - did you solve the CORS issue before re-enabling?

File: turbo.json [hunk 4]
```diff
@@ -18,7 +19,6 @@
     "lint": {},
     "check-types": {},
     "dev": {
-      "dependsOn": ["^build"],
```
Comment (callicles) [inline turbo.json:21]: Why did you remove this? that used to be what made the glue work.

File: apps/create-moose-app/tsconfig.json (hunk 1 above)
Comment (tg339) [reply apps/create-moose-app/tsconfig.json:2]: make it clear in the docs what's coming from the repo dependencies

File: turbo.json (hunk 4 above)
Comment (tg339) [reply turbo.json:21]: Not relevant for an internal dependency

**Review by georgevanderson — APPROVED** [review]
:shipit:
